### PR TITLE
Add resourceGroup as a tag in the emitted metrics

### PR DESCRIPTION
**Pull request by dlmarion**
Related to #3991

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java [hunk 1]
```diff
@@ -78,7 +79,7 @@ private static void initializeMetrics(boolean enabled, boolean jvmMetricsEnabled
 
       List<Tag> tags = new ArrayList<>();
       tags.add(Tag.of("process.name", processName));
-
+      tags.add(Tag.of("resource.group", resourceGroup));
```
Comment (dlmarion) [inline core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java:82]: wondering if we should add instance_name to the tags...

Comment (EdColeman) [reply core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java:82]: If I recall correctly, there are tags that are automatically added on initialization - maybe we could make that a default?  I'm not sure of the impact of adding a "constant" tag (assuming most installations run a single instance). Or maybe instance name is synonymous with system name?

If metrics are forwarded to a central collector then we should support a tag that allows differentiation, so the usages may be wider than running multiple instances within a single cluster,

Comment (EdColeman) [reply core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java:82]: Likely outside of the scope of this PR, but maybe we could consider migrating the standard tags to something that mirrors the syslog naming conventions (https://datatracker.ietf.org/doc/html/rfc5424)

That could make it easier to correlate metrics and log messages that are sent to syslog.

Comment (dlmarion) [reply core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java:82]: added instance name tag in #3998 targeting 2.1.3

**Review by EdColeman — APPROVED** [review]

**Pull request merged**